todo_app/templates/index.html

```diff
@@ -11,8 +11,24 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+                <li>
+                    {{ item.title }}
+                </li>
+            {% endfor %}
       </ul>
     </div>
+
+  <form action="/add" method="post" class="add-item">
```

Comment:
You can use url_for in the views too:

[assuming you have renamed the login function to add_item]
```suggestion
  <form action="{{ url_for("add_item") }}" method="post" class="add-item">
```